test/integration/expression-tests/image/basic/test.json

```diff
@@ -17,7 +21,8 @@
       "type": "resolvedImage"
     },
     "outputs": [
-      {"name":"monument-15","available":true}
+      {"name":"monument-15","available":true},
+      {"name":"monument-15","available":false}
```

Comment:
It might be less confusing to use a different name for the false test.

Reply:
I'm not sure if I understand exactly what you mean, are you suggesting I separate this test into two different files to use a different image name?

Reply:
No, it's a nit change. I was just thinking that something like `{"name": "foo", "available": false}` would be a little more obvious when casually looking at the test.

Reply:
I think you're suggesting something like this?
```js
"expressions":
  [
    ["image", "monument-15"],
    ["image", "foo"]
  ],
"inputs": [
    [ {"availableImages": ["monument-15"]}, {} ]
  ],
```
This doesn't seem to work. It seems to me that each test can only take one expression (though arbitrarily many "inputs"). So to test two distinct expressions we'd need two different test files.

Please correct me if I'm wrong, I agree that your syntax would be elegant!

Reply:
Actually, looking at this again, this is good. I retract my nit.